includes/Classifai/Providers/Azure/Speech.php

```diff
@@ -213,23 +215,7 @@ public function connect_to_service( array $args = array() ): array {
 			$default['endpoint_url']
 		);
 
-		if ( function_exists( 'vip_safe_wp_remote_get' ) ) {
-			$response = vip_safe_wp_remote_get(
-				$request_url,
-				'',
-				3,
-				1,
-				20,
-				$request_params
-			);
-		} else {
-			$request_params['timeout'] = 20; // phpcs:ignore WordPressVIPMinimum.Performance.RemoteRequestTimeout.timeout_timeout
-			// phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get -- use of `vip_safe_wp_remote_get` is done when available.
-			$response = wp_remote_get(
-				$request_url,
-				$request_params
-			);
-		}
+		$response = safe_wp_remote_get( $request_url, $request_params );
```

Comment:
So previously we were setting a higher timeout value here to give the request longer to succeed. That is now removed and I think we'll want to add that back.